druid/src/event.rs

```diff
@@ -14,12 +14,13 @@
 
 //! Events.
 
-use crate::kurbo::{Rect, Shape, Size, Vec2};
+use crate::kurbo::{Rect, Size};
+use std::ops::{Add, Sub};
```

Comment:
The (currently undocumented) standard of ordering imports is std/external/internal. Given that, please move the `use crate::kurbo::{Rect, Size};` line below, next to the other `use crate::...` lines. Yes it was wrong before too.